internal/manifests/manifests.go

```diff
@@ -353,6 +353,10 @@ func (m *Manifests) validateAllowedToModifyManifests(ctx context.Context, cluste
 }
 
 func (m *Manifests) validateUserSuppliedManifest(ctx context.Context, clusterID strfmt.UUID, manifestContent []byte, fileName string) error {
+	maxFileSizeBytes := 1048576
```

Comment:
```suggestion
	maxFileSizeBytes := 1024 * 1024 * 1024
```

Reply:
I think this should be 

`maxFileSizeBytes := 1024 * 1024`